src/client/pfs/pfs.proto

```diff
@@ -624,12 +624,12 @@ service API {
   rpc DeleteAll(google.protobuf.Empty) returns (google.protobuf.Empty) {}
   // Fsck does a file system consistency check for pfs
   rpc Fsck(FsckRequest) returns (stream FsckResponse) {}
-
-  // RPCs specific to Pachyderm 2.
-  rpc PutTarV2(stream PutTarRequestV2) returns (google.protobuf.Empty) {}
-  rpc GetTarV2(GetTarRequestV2) returns (stream google.protobuf.BytesValue) {}
+  
+  // V2 methods
```

Comment:
Would you change the comment back to the original. It is more descriptive and is a pattern I am following elsewhere (* specific to Pachyderm 2).